### PR TITLE
Initial Mainframe styles

### PR DESCRIPTION
**Pull request by danielledeleo**
This PR adds mostly typographic styles to the new theme. It can be safely merged into the main branch and toggled with a console command: `useNewTheme(true)`. 
Screenshot 2024-10-01 at 4 23 12 PM

### CODE REVIEW

**Review by nginx-jack — APPROVED** [review]

File: layouts/partials/styles.html [hunk 1]
```diff
@@ -11,6 +11,11 @@
 <link href="{{ $css7.RelPermalink }}" integrity="{{ $css7.Data.Integrity }}" rel="stylesheet" type="text/css" id="css7"
   disabled>
 
+<link rel="preconnect" href="https://rsms.me/">
+<link rel="stylesheet" href="https://rsms.me/inter/inter.css">
```
Comment (jputrino) [inline layouts/partials/styles.html:15]: Do we need permission or some type of review to be able to use this project?

Our best practices are to download from source, include a copy in the repo, and load it using the hugo data integrity functions as above. That helps safeguard against risks to the source being compromised. One that's done, I'm happy to merge this in.

**Review by jputrino — CHANGES_REQUESTED** [review]

File: layouts/partials/styles.html (hunk 1 above)
Comment (nginx-jack) [reply layouts/partials/styles.html:15]: It's fine licence wise, it's the standard OFL https://fonts.google.com/specimen/Inter/license.
I'll pull it directly into the repo alright, that's a good shout 👍

**Review by ADubhlaoich — APPROVED** [review]
I think it would be nice to have `system-ui` as a fallback in the font stack but small LOGAF.

**Comment by nginx-jack** [comment]
> I think it would be nice to have `system-ui` as a fallback in the font stack but small LOGAF.

No time like the present!

**Review by jputrino — APPROVED** [review]

**Pull request merged**